sdk/ml/azure-ai-ml/azure/ai/ml/entities/_job/automl/image/image_classification_job.py

```diff
@@ -37,6 +37,15 @@ def __init__(
 
         :param primary_metric: The primary metric to use for optimization
         :param kwargs: Job-specific arguments
+
+        .. admonition:: Example:
+
+            .. literalinclude:: ../samples/ml_samples_automl_image.py
+                :start-after: [START automl.image_classification_job]
```

Comment:
the corresponding example is tagged with `START automl.automl_image_job.image_classification_job` in the samples file so no example is displayed in the docs currently.

Reply:
Great catch. Added